config.go

```diff
@@ -60,23 +60,28 @@ type config struct {
 	CORSOrigin string `long:"corsorigin" description:"The value to send in the Access-Control-Allow-Origin header. Header will be omitted if empty."`
 }
 
-// loadConfig starts with a skeleton default config, and reads in user provided
-// configuration from the command line. It does not provide a full set of
-// defaults or validate user input because validation and sensible default
-// setting are performed by the lndclient package.
-func loadConfig() (*config, error) {
-	// Start with a default config.
-	config := &config{
+// DefaultConfig returns all default values for the config struct.
+func DefaultConfig() config {
+	return config{
 		RPCServer:        defaultRPCHostPort,
 		network:          defaultNetwork,
 		MacaroonFile:     defaultMacaroon,
 		MinimumMonitored: defaultMinimumMonitor,
 		DebugLevel:       defaultDebugLevel,
 		RPCListen:        defaultRPCListen,
 	}
+}
+
+// LoadConfig starts with a skeleton default config, and reads in user provided
+// configuration from the command line. It does not provide a full set of
+// defaults or validate user input because validation and sensible default
+// setting are performed by the lndclient package.
```

Comment:
Just a question about not validating. Once we start adding more values to config here which are not lndclient relevant (something like `minimum_forward_rate` for closing channels that are inactive), where would validation those config settings live?

Reply:
Once the need for faraday specific config arises, I think we should probably do it in a similar way as it's done in `lnd` with this PR: https://github.com/lightningnetwork/lnd/pull/4248.
Export a `ValidateConfig` function that can be called after `LoadConfig` to check and sanitize the provided values.